lib/spack/spack/cmd/repo.py

```diff
@@ -132,6 +132,33 @@ def setup_parser(subparser: argparse.ArgumentParser):
         help="automatically migrate the repository to the latest Package API",
     )
 
+    # Update
+    update_parser = sp.add_parser("update", help=repo_update.__doc__)
+    update_parser.add_argument(
+        "namespaces", nargs="*", default=[], help="repository namespaces to update"
```

Comment:
in #50875 we have `--names` (repos.yaml key) vs `--namespaces` (repo.yaml name)

This should probably be just `name` then, since you cannot update a single namespace from a git monorepo that has multiple package repos.